### DISCUSSION TITLE
部署完成后提示部署错误 404

### DISCUSSION THREAD

**Discussion by ZGQ-inc** [spencerwooo/onedrive-vercel-index#475] [Q&A] [answered]
访问页面提示404

错误信息：
`{"error":{"code":"504","message":"An error occurred with your deployment"}}`

这种情况是不是需要`设定REDIS_URL 环境变量`？

**Comment by myl7** [comment, marked as answer]
先看一下 Vercel deployment 成功与否，以及相应的 log 吧
